sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -2296,11 +2297,11 @@ class RapidsConf(conf: Map[String, String]) extends Logging {
 
   lazy val perTaskOverhead: Long = get(TASK_OVERHEAD_SIZE)
 
-  lazy val concurrentGpuTasks: Int = get(CONCURRENT_GPU_TASKS)
```

Comment:
Why is this deleted? There were also several others deleted. If they are not used I understand, but at the same time I would prefer that to be in a separate PR.

Reply:
It is unused, but understand the point of a separate PR